distributed/config.py

```diff
@@ -64,6 +64,12 @@ def load_config_file(config, path):
 
 
 def load_env_vars(config):
+    """
+    Scan environment variables for `DASK_`.
```

Comment:
See http://dask.pydata.org/en/latest/develop.html#docstrings for style conventions